src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts

```diff
@@ -24,16 +30,24 @@ export class ServicesPageHelper extends PageHelper {
     return this.selectOption('service_type', serviceType);
   }
 
+  @PageHelper.restrictTo(pages.index.url)
+  clickServiceTab(serviceName: string, tabName: string) {
+    this.getExpandCollapseElement(serviceName).click();
+    cy.get('cd-service-details').within(() => {
+      this.getTab(tabName).click();
+    });
+  }
+
   @PageHelper.restrictTo(pages.create.url)
-  addService(serviceType: string, exist?: boolean) {
+  addService(serviceType: string, exist?: boolean, count = '1') {
```

Comment:
`count  = '1` : is this to avoid waiting for daemons running?

Reply:
nope. for testing force maintenance check I need to deploy atleast 2 rgw service. But earlier the count was hardcoded to the code. Now I used a variable to control that. @avanthakkar